drivers/volume/portworx/portworx.go

```diff
@@ -1900,6 +1900,29 @@ func (d *portworx) ValidateVolumeInPxctlList(volumeName string) error {
 	return nil
 }
 
+func (d *portworx) UpdateFBDANFSEndpoint(volumeName string, newEndpoint string) error {
+	nodes := node.GetStorageDriverNodes()
+	cmd := fmt.Sprintf("%s --pure_nfs_endpoint %s %s", pxctlVolumeUpdate, newEndpoint, volumeName)
+	_, err := d.nodeDriver.RunCommandWithNoRetry(
+		nodes[0],
+		cmd,
+		node.ConnectionOpts{
+			Timeout:         crashDriverTimeout,
+			TimeBeforeRetry: defaultRetryInterval,
+		})
+	if err != nil {
+		return fmt.Errorf("failed setting FBDA NFS endpoint for volume '%s' to '%s', Err: %v", volumeName, newEndpoint, err)
```

Comment:
Can we also include node name or ID in the log message?

Reply:
Done